.github/workflows/athena_test.yml

```
@@ -41,3 +41,24 @@
+      if: always()
+      uses: mikepenz/action-junit-report@v5.5.0
+      id: test-report
+      with:
+        report_paths: 'athena/test-results/*.xml'
+        check_name: "Athena Test Report"
+        fail_on_failure: true
+        require_tests: true
+        require_passed_tests: true
+        detailed_summary: true
+        include_time_in_summary: true
+        group_suite: true    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)